nevergrad/optimization/mutations.py

```diff
@@ -15,15 +17,21 @@ class Mutator:
     def __init__(self, random_state: np.random.RandomState) -> None:
         self.random_state = random_state
 
-    def doerr_discrete_mutation(self, parent: tp.ArrayLike) -> tp.ArrayLike:
+    def other_random(self, v: float, arity: int):
```

Comment:
this name is not clear, can you clarify? maybe add a docstring, + a test

Reply:
done!